src/components/ar-paint-controls.js

```diff
@@ -0,0 +1,155 @@
+AFRAME.registerSystem('ar-paint-controls', {
+  numberStrokes: 0
+});
+
+/* globals AFRAME THREE */
+AFRAME.registerComponent('ar-paint-controls', {
+  dependencies: ['brush', 'raycaster'],
+
+  init: function () {
+    var el = this.el;
+    this.controller = null;
+
+    this.size = el.sceneEl.renderer.getSize();
+    this.pointer = new THREE.Vector2();
+    // normalized device coordinates position
+    this.pointerNdc = new THREE.Vector2();
+
+    this.numberStrokes = 0;
```

Comment:
why do you need this? is not used anywhere else right?

Reply:
It isn't used. I have it because was included on paint-controls.js too. I delete it on the next commit.